keps/sig-node/3386-kubelet-evented-pleg/README.md

```diff
@@ -348,6 +349,9 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - Add E2E Node Conformance presubmit job in CI 
 - Add E2E Node Conformance periodic job in CI
 
+#### GA
+- Add a stress test in e2e for Evented PLEG. This feature should not be enabled by default without having such a test.
```

Comment:
Can you expand on specific tests that needs to be implemented under the e2e section. Stress is a wide term. I think we need to list dimensions of stress - a lot of containers, a lot of events from a single container, maybe what type of events needs to be tested carefully.

Another aspect of testing - kubelet restart scenario while container state is being changed rapidly. Events when API server is down, etc.

Reply:
I agree it makes sense to add that in the e2e section. I will update the PR soon with that info.